readme/the-dev-environment.md

```diff
@@ -13,11 +13,21 @@ Only interact with protocol addresses shipped with the client as these are verif
 
 ## Test Markets
 
-Currently there is no admin client to allow for developers to create their own markets. Until this client is shipped, markets will be regularly added to the protocol to enable development. Similarly, the token used to enter the market can be obtained via an airdrop whilst we work on a faucet.
+Currently there is no admin client to allow for developers to create their own markets. As such, market automation is in place to facilitate development:
+
```

Comment:
Can we state somewhere that an admin client is on its way to allow developers to create their own markets?

Reply:
oh yeah, I can't believe I made that _less clear_